admin/tabs/userData/roles/RoleModel.ts

```diff
@@ -38,12 +38,16 @@ export class RoleModel extends HoistModel {
     @observable.ref allRoles: HoistRole[] = [];
     @observable.ref moduleConfig: RoleModuleConfig;
 
+    @bindable groupingDisabled = false;
```

Comment:
its not that grouping is 'disabled'  its that its not actually being grouped at the moment.  Would call it -- `showInGroups' or something like that